microsoft-edge/extensions-chromium/publish/reply-to-user-review.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Respond to user reviews about your extension
+description: Developers can view their Microsoft Edge extension’s reviews and ratings and respond to user reviews from the Partner Center dashboard.
```

Comment:
```suggestion
description: Developers can view their Microsoft Edge extension's reviews and ratings and respond to user reviews from the Partner Center dashboard.
```
https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#apostrophes-and-quotation-marks